infra/app-flask/service/document_data_extraction.tf

```
@@ -0,0 +1,56 @@
+locals {
+  document_data_extraction_config = local.environment_config.document_data_extraction_config
+
+  # convert tags map to bedrock data automation format
+  bda_tags_dict = [
+    for key, value in local.tags : {
+      key   = key
+      value = value
+    }
+  ]
+
+  document_data_extraction_environment_variables = local.document_data_extraction_config != null ? {
+    DDE_INPUT_BUCKET_NAME  = local.document_data_extraction_config.input_bucket_name
+    DDE_OUTPUT_BUCKET_NAME = local.document_data_extraction_config.output_bucket_name
+    DDE_PROJECT_ARN        = module.dde[0].bda_project_arn
+  } : {}
+}
+
+module "dde_input_bucket" {
+  count        = local.document_data_extraction_config != null ? 1 : 0
+  source       = "../../modules/storage"
+  name         = local.document_data_extraction_config.input_bucket_name
+  is_temporary = local.is_temporary
+}
+
+module "dde_output_bucket" {
+  count        = local.document_data_extraction_config != null ? 1 : 0
+  source       = "../../modules/storage"
+  name         = local.document_data_extraction_config.output_bucket_name
+  is_temporary = local.is_temporary
+}
+
+module "dde" {
+  count  = local.document_data_extraction_config != null ? 1 : 0
+  source = "../../modules/document-data-extraction/resources/bedrock-data-automation"
+
+  bda_standard_output_configuration = local.document_data_extraction_config.bda_standard_output_configuration
+  bda_tags                          = local.bda_tags_dict
+
+  blueprints_map = {
+    for blueprint in local.document_data_extraction_config.enabled_blueprints :
+    split(".", blueprint)[0] => {
+      schema                 = file("${local.document_data_extraction_config.blueprints_path}/${blueprint}")
+      type                   = "DOCUMENT"
+      kms_encryption_context = null
+      kms_key_id             = null
+      tags                   = local.bda_tags_dict
+    }
+  }
+
+  name_prefix = local.document_data_extraction_config.name
```

Couple things.

To resolve the CI resource creation issues, any resource names in here need to have `local.prefix` added to them for uniqueness in the PR/temporary environments, like:
```suggestion
  name_prefix = "${local.prefix}${local.document_data_extraction_config.name}
```
(e.g. https://github.com/navapbc/template-infra/blob/84686bfd79aa69acbac9cae0e96c5286fb40fa93/infra/%7B%7Bapp_name%7D%7D/service/notifications.tf#L13)

And `name_prefix` can just be `name` like most (all?) of the other modules.